### PR TITLE
[no ticket][risk=low] increase egress bypass window description field max size

### PR DESCRIPTION
**Pull request by yonghaoy**
Supports had issue when using long description., increase size limit to higher number

---
**PR checklist**

- [ ] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have manually tested this change and my testing process is described above.
- [ ] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [ ] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [ ] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [ ] None of the above apply to this change.

### CODE REVIEW

File: api/db/changelog/db.changelog-223-modify-egress-bypass-window-datatype.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="yonghao" id="db.changelog-223-modify-egress-bypass-window-datatype">
+    <modifyDataType
+      columnName="description"
+      newDataType="varchar(8128)"
```
Comment (NehaBroad) [inline api/db/changelog/db.changelog-223-modify-egress-bypass-window-datatype.xml:10]: whats the significance of 8128?

**Comment by jmthibault79** [comment]
was the change to `aou-utils` intentional here?

File: api/db/changelog/db.changelog-223-modify-egress-bypass-window-datatype.xml (hunk 1 above)
Comment (yonghaoy) [reply api/db/changelog/db.changelog-223-modify-egress-bypass-window-datatype.xml:10]: Seems the access reason will be large text and we want to store that in our DB. So increase to 8128. Should be good for now

Comment (yonghaoy) [reply api/db/changelog/db.changelog-223-modify-egress-bypass-window-datatype.xml:10]: I learnt this from this PR comment: https://github.com/DataBiosphere/leonardo/pull/3293#discussion_r1170539130

**Review by NehaBroad — APPROVED** [review]

File: api/db/changelog/db.changelog-223-modify-egress-bypass-window-datatype.xml (hunk 1 above)
Comment (NehaBroad) [reply api/db/changelog/db.changelog-223-modify-egress-bypass-window-datatype.xml:10]: I just realized in UI the MAX field the description field is 4000 https://github.com/all-of-us/workbench/blob/050b625bb0f9565f021090bb640333cff26b60b5/ui/src/app/pages/admin/user/admin-user-egress-bypass.tsx#L20C1-L20C37
, do we want to change that as well?

Comment (jmthibault79) [reply api/db/changelog/db.changelog-223-modify-egress-bypass-window-datatype.xml:10]: Yes we need to keep these 2 values in agreement.

@yonghaoy is 4000 large enough?  It's not clear why 8128 was chosen here.

Comment (yonghaoy) [reply api/db/changelog/db.changelog-223-modify-egress-bypass-window-datatype.xml:10]: ah. okay. I was thinking backend usually has a larger limit than FE. But 4000 also SG. Changed to 4000.

**Pull request merged**